src/Microsoft.OpenApi.Hidi/Program.cs

```diff
@@ -78,13 +79,16 @@ static async Task<int> Main(string[] args)
             };
 
             transformCommand.SetHandler<string, string, FileInfo, bool, string?, OpenApiFormat?, LogLevel, bool, bool, string, string, string, CancellationToken> (
-                OpenApiService.ProcessOpenApiDocument, descriptionOption, csdlOption, outputOption, versionOption, formatOption, logLevelOption, inlineOption, resolveExternalOption, filterByOperationIdsOption, filterByTagsOption, filterByCollectionOption);
+                OpenApiService.TransformOpenApiDocument, descriptionOption, csdlOption, outputOption, cleanOutputOption, versionOption, formatOption, logLevelOption, inlineOption, resolveExternalOption, filterByOperationIdsOption, filterByTagsOption, filterByCollectionOption);
 
             rootCommand.Add(transformCommand);
             rootCommand.Add(validateCommand);
 
             // Parse the incoming args and invoke the handler
-            return await rootCommand.InvokeAsync(args);
+            await rootCommand.InvokeAsync(args);
+
+            //// Wait for logger to write messages to the console before exiting
+            await Task.Delay(10);
```

Comment:
This delay was necessary to ensure there was time for log messages to get written to the console.